docs/contributing/database-migrations/edd.mdx

```diff
@@ -190,6 +190,13 @@ Create a PR moving the future scripts.
 - `DbScripts_future` to `DbScripts`, prefix the script with the current date, but retain the
   existing date.
 - `dbo_future` to `dbo`.
+  <bitwarden>
+    <li>
```

Comment:
I assume we need the `li` here and `-` doesn't work?

Reply:
yeah... not super happy with it but the `\n`s needed messed up the markdown spacing or failed to compile due to trying to `</li>` in the middle of the `<bitwarden>` element 😭